CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.17.0] - 2022-02-04
+
+### Added
```

Comment:
Nit: there's some inconsistency over "do we have a newline between the title and list" in this doc; let's add a newline here

Reply:
Good catch, fixed in the latest revision